Clickstream/EventScheduler/Core/EventBatchCreator.swift

```diff
@@ -69,3 +70,18 @@ extension DefaultEventBatchCreator {
         networkBuilder.isAvailable
     }
 }
+
+
+extension DefaultEventBatchCreator {
+    private func trackHealthEvents(batch: EventBatch, events: [Event]) {
+        #if TRACKER_ENABLED
+        if events.first?.type != TrackerConstant.HealthEventType {
```

Comment:
if event is not instant event check is missing here, is that intended?

Reply:
Nice catch, thanks will add it.